doc: Added release 9.0 notes

rkothiya opened the pull request:
Updates: #1868

Change-Id: I8a49ab7ccfd45bd3b469ce34b6625b99cbfbb329
Signed-off-by: Rinku Kothiya <rkothiya@redhat.com>

Commit:
doc: Added release 9.0 notes

Updates: #1868

Change-Id: I8a49ab7ccfd45bd3b469ce34b6625b99cbfbb329
Signed-off-by: Rinku Kothiya <rkothiya@redhat.com>

msaju commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,258 @@
+1. Releases that receive maintenance updates post release 9 is 8 
+([reference](https://www.gluster.org/release-schedule/))
+
+2. Release 8 will receive maintenance updates around the 30th of every month
```
release 9

Shwetha-Acharya reviewed (commented):
release 8 or release 9?

Shwetha-Acharya commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,263 @@
+1. Releases that receive maintenance updates post release 9 is 8 
+([reference](https://www.gluster.org/release-schedule/))
+
+2. Release 9 will receive maintenance updates around the 30th of every month
```
` Release 9 will receive maintenance updates around the 30th of every month` release 8 or 9?

msaju replied:
In release note of 9, we should be talking about release 9.0 only....So further maintenance releases on 30th of every month. Is there anything I missed?.

Shwetha-Acharya replied:
@msaju Sorry, I got it now

Shwetha-Acharya commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,263 @@
+([reference](https://www.gluster.org/release-schedule/))
+
+2. Release 9 will receive maintenance updates around the 30th of every month
+for the first 3 months post release (i.e Jan'21, Feb'21, Mar'21). Post the
```
just confused as at line 18, it mentions 30th of every month

msaju replied:
Its the month and Year....line 18 is the date.

mykaul commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,263 @@
+
+- Added support for:
+
+  - IO Uring in Gluster
```
```suggestion
  - io_uring in Gluster
```

rkothiya replied:
Done

mykaul commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,263 @@
+- Added support for:
+
+  - IO Uring in Gluster
+  - 5K volumes 
```
```suggestion
  - support running with up to 5000 volumes 
```

rkothiya replied:
Done

mykaul commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,263 @@
+  - IO Uring in Gluster
+  - 5K volumes 
+
+- Several fixes addressing
```
this is not a highlight?

rkothiya replied:
Done, removed it.

mykaul commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,263 @@
+
+- Several fixes addressing
+
+  - Code optimization and improvements, bugs and other identified fixes from earlier 8.x releases
```
same?

rkothiya replied:
Done, I have removed it.

mykaul commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,263 @@
+- [#1782](https://github.com/gluster/glusterfs/issues/1782) Rebalance is reporting status twice upon stopping, resulting ...
+- [#1778](https://github.com/gluster/glusterfs/issues/1778) volume set: failed: ganesha.enable is already 'off'.
+- [#1775](https://github.com/gluster/glusterfs/issues/1775) core: lru_size showing -1 with zero inodes in the list in the...
+- [#1772](https://github.com/gluster/glusterfs/issues/1772) build: add LTO as a configure option
```
this one is worth highlighting under Features, IMHO.

rkothiya replied:
Done

mykaul commented on doc/release-notes/9.0.md:
```diff
@@ -0,0 +1,263 @@
+- [#1226](https://github.com/gluster/glusterfs/issues/1226) Gluster webhook update throws error
+- [#1225](https://github.com/gluster/glusterfs/issues/1225) fuse causes glusterd to dump core
+- [#1223](https://github.com/gluster/glusterfs/issues/1223) Failure of tests/basic/gfapi/gfapi-copy-file-range.t
+- [#1222](https://github.com/gluster/glusterfs/issues/1222) [RFE] Improve rebalance of sparse files
```
this one could be mentioned as a nice feature.

rkothiya replied:
Done

Commit:
doc: Added release 9.0 notes

Updated the highlights and features section according to the
review comments.

Updates: #1868

Change-Id: I3fa89e9f186e15a074cf13a14e711e572df10886
Signed-off-by: Rinku Kothiya <rkothiya@redhat.com>

Commit:
doc: Added release 9.0 notes

Updated the dates of maintenance update.

Updates: #1868

Change-Id: I4049e019312d951eb68f7b68914b265bfe74e2e4
Signed-off-by: Rinku Kothiya <rkothiya@redhat.com>

msaju reviewed (commented):
Looks good...

msaju reviewed (approved):
LGTM

rkothiya commented:
/run regression

Sheetalpamecha reviewed (approved):
LGTM
